server/models/Expense.js

```diff
@@ -194,7 +194,7 @@ export default function(Sequelize, DataTypes) {
       CollectiveId: this.collective.id,
       data: {
         host: get(host, 'minimal'),
-        collective: this.collective.minimal,
+        collective: { ...this.collective.minimal, isActive: this.collective.isActive },
```

Comment:
can we add `isActive` to minimal instead?

Reply:
Do we really want to "pollute" the `minimal` used elsewhere with this information that is only needed here?

Reply:
I don't know the history behind it. Looks weird to extend it with just one property.

Now that I'm looking at it, I'm wondering why the following are `minimal`:

```
twitterHandle,
githubHandle,
publicUr,
mission,
isSupercollective,
```

Reply:
This entire thing is bogus. It was initially meant for storing a minimum set of data for the activity table. But we should revisit that at some point.